flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/Climb.flt

```diff
@@ -145,7 +145,7 @@ A32NX_OVHD_ADIRS_ADR_3_PB_IS_ON=1
 A32NX_OVHD_PRESS_MODE_SEL_PB_IS_AUTO = 1
 A32NX_OVHD_PRESS_MAN_VS_CTL_SWITCH = 1
 A32NX_TRANSPONDER_MODE = 1
-A32NX_TRANSPONDER_ALT_RPTG = 1
```

Comment:
Please keep this `A32NX_TRANSPONDER_ALT_RPTG` and fix the changes that the tcas PR made.